wire-ios-data-model/Tests/Source/Utils/MockProteusProvider.swift

```diff
@@ -48,16 +46,21 @@ class MockProteusProvider: ProteusProviding {
         }
     }
 
-    var mockCanPerform = true
-    var canPerform: Bool {
+    public var mockCanPerform = true
+    public var canPerform: Bool {
         return mockCanPerform
     }
-}
-
-extension XCTestCase {
 
-    func spyForTests() -> SpyUserClientKeyStore {
-        let url = self.createTempFolder()
+    public static func spyForTests() -> SpyUserClientKeyStore {
+        let url = Self.createTempFolder()
         return SpyUserClientKeyStore(accountDirectory: url, applicationContainer: url)
     }
+
+    // FIXME: [F] this is defined in WireTesting, somehow it is not possible to import here for now
```

Comment:
You need to link the WireTesting framework if you want to import code from it.

<img width="766" alt="Screenshot 2023-12-12 at 10 37 00" src="https://github.com/wireapp/wire-ios/assets/15628584/d1dd66c6-fd57-458a-a9df-9d892a835ae5">

Reply:
I tried but had other issue with XCTest then

Reply:
There is a non-static method `createTempFolder()` defined in an extension to `XCTest`, btw.

Reply:
added a ticket to fix this later